Resources/Prototypes/_Sunrise/Surgery/amputation_steps.yml

```
@@ -16,3 +16,70 @@
+  - type: SurgeryStepBleedEffect
+    damage:
+      types:
+        Slash: 5
+  - type: SurgeryStepEmoteEffect
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepLigateVessels
+  name: Ligate Vessels
+  components:
+    - type: SurgeryStep
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)